server/athenian/api/controllers/default_controller.py

```diff
@@ -1,21 +1,41 @@
-from typing import List, Dict
+import datetime
+
 from aiohttp import web
+from sqlalchemy import select, sql
 
+from athenian.api import FriendlyJson
+from athenian.api.models import CalculatedMetric, CalculatedMetricValues
 from athenian.api.models.calculated_metrics import CalculatedMetrics
-from athenian.api.models.invalid_request_error import InvalidRequestError
+from athenian.api.models.db import github
+# from athenian.api.models.invalid_request_error import InvalidRequestError
 from athenian.api.models.metrics_request import MetricsRequest
-from athenian.api.models.no_source_data_error import NoSourceDataError
-from athenian.api import util
+# from athenian.api.models.no_source_data_error import NoSourceDataError
 
 
 async def calc_metrics(request: web.Request, body) -> web.Response:
```

Comment:
lets split this function into sth. like `fetch repos`, `collect metrics`

Reply:
I totally agree. This is currently a dummy shitty code to demo that everything works together. Let me start working on real logic in the next PR.